src/sass/modules/_m-vet-nav.scss

```diff
@@ -433,16 +432,16 @@ body.va-pos-fixed {
   white-space: nowrap;
 }
 
-[id=search] {
+[id="search"] {
   width: 30rem;
 
-  [type=text] {
+  [type="text"] {
     margin: 0;
     flex: 1 0 20rem;
   }
 
-  [type=submit] {
-    border-radius: 0 .3rem .3rem 0;
+  [type="submit"] {
+    border-radius: 0 0.3rem 0.3rem 0;
     display: flex;
```

Comment:
I think your editor may have autoformatted some of the values here. No big deal in this PR, but just a heads up for in the future if you want to keep a file diff small or a branch comparison focused to a single thing.

Reply:
got it, i'll fix my editor settings to better fix this issue from happening in the future

Reply:
It's actually a good update to the file, it can just make things a little fuzzy sometimes when another dev looks at the changes. Either way, good work!